run/map/watermark/guideposts.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "assetName":"test.png",
+    "src":"/static/run/map/watermark/test.png",
+    "imageRef":"watermark",
+    "srcX":0,
+    "srcY":0,
+    "srcHeight":634,
+    "srcWidth":634,
+    "dWidth":200,
+    "dHeight":200,
+    "mapCoord":{
+      "mapName":"world1",
+      "x":10,
+      "y":4
+    },
```

Comment:
I think `mapCoord` should be an array of map coordinates. In this way, this watermark is reusable.
```
mapCoord: [
  {"mapName": "world1", "x": 10, "y": 4},
  {"mapName": "world1", "x": 5, "y": 15},
  {"mapName": "world2", "x": 6, "y": 6}
]
```